sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -3349,20 +3385,20 @@ class Dataset[T] private[sql](
     }
   }
 
-  /** Convert to an RDD of ArrowPayload byte arrays */
-  private[sql] def toArrowPayload(plan: SparkPlan): RDD[ArrowPayload] = {
+  /** Convert to an RDD of serialized ArrowRecordBatches. */
+  private[sql] def getArrowBatchRdd(plan: SparkPlan): RDD[Array[Byte]] = {
```

Comment:
@BryanCutler, not a big deal at all but how about `to` like the before? This reminds me of QueryExecution's toRdd and the previous one looks slightly better.

Reply:
Yeah, I can't remember why I changed it.. but I think you're right it so I'll change it back.